docs/content/about-iota/iota-wallet/getting-started.mdx

```diff
@@ -85,3 +85,91 @@ To use your IOTA Wallet on different devices or browsers, you can import it usin
 By following these steps, you can manage, stake,
 and transfer your digital assets on the IOTA network from any of your devices, ensuring a seamless experience.
 
+## Use the Balance Finder
+
+[The way accounts and wallets are managed in the IOTA Wallet is different than in Firefly and similar Stardust Wallets](#whats-changed).
+You can use the IOTA balance finder to import all your Firefly wallets and unify them into a single profile. 
+
+### How to Access the Balance Finder
+
+If you are importing a profile, you will be prompted to use the Balance Finder as part of the import process.
+
+   ![Balance Finder in Prompt](/img/about-iota/iota-wallet/getting-started/access-balance-finder-prompt.png)
+
+You can also access the balance finder by clicking on the meatball menu (`...`) beside the desired account dropdown in the
+`Manage Accounts` section.
+
+   ![Balance Finder in Accounts View](/img/about-iota/iota-wallet/getting-started/access-balance-finder.png)
+
+### Find and Unify Balances
```

Comment:
I would call it only Find Balances, you cant unify them unless you send the funds from one address to another which is a manual process